plugins/synaptics-cape/fu-synaptics-cape-device.c

```diff
@@ -14,10 +14,11 @@
 #include "fu-synaptics-cape-firmware.h"
 
 /* defines timings */
-#define FU_SYNAPTICS_CAPE_DEVICE_USB_CMD_WRITE_TIMEOUT	20000 /* us */
-#define FU_SYNAPTICS_CAPE_DEVICE_USB_CMD_READ_TIMEOUT	30000 /* us */
+#define FU_SYNAPTICS_CAPE_DEVICE_USB_CMD_WRITE_TIMEOUT	20    /* ms */
+#define FU_SYNAPTICS_CAPE_DEVICE_USB_CMD_READ_TIMEOUT	30    /* ms */
 #define FU_SYNAPTICS_CAPE_DEVICE_USB_CMD_RETRY_INTERVAL 10    /* ms */
 #define FU_SYNAPTICS_CAPE_DEVICE_USB_CMD_RETRY_TIMEOUT	300   /* ms */
+#define FU_SYNAPTICS_CAPE_DEVICE_USB_CMD_INTR_TIMEOUT	300000 /* ms */
```

Comment:
Usually, device can respond the fw update request within 300 ms. I think it could be the FW was too busy to respond the request. According to FW API spec, the maximum of timeout here should be 2 seconds.